src/packaging/bin/cassandra-reaper

```diff
@@ -14,7 +14,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-REAPER_JAR=$(find . -maxdepth 4 -regex '.*/cassandra-reaper-.*[0-9rT]\.jar')
+REAPER_JAR=$(find /usr/share/cassandra-reaper/ -maxdepth 4 -regex '.*/cassandra-reaper-.*[0-9rT]\.jar')
```

Comment:
The idea here was to use the reaper found under the current directory.

Using standard installations of Reaper should be done below, eg where we are looking under `/usr/local/share`

Reply:
in standrat installation systemd try run find in "/" root directory because of https://www.freedesktop.org/software/systemd/man/systemd.exec.html#WorkingDirectory=

Reply:
**Find** in an explicitly specified directory is much more understandable and will not allow problems that I encountered in #800 
or maybe we need specified the WorkingDirectory directive in https://github.com/thelastpickle/cassandra-reaper/blob/master/src/packaging/debian/cassandra-reaper.service

Reply:
> or maybe we need specified the WorkingDirectory directive in https://github.com/thelastpickle/cassandra-reaper/blob/master/src/packaging/debian/cassandra-reaper.service

Yes, that would be a correct solution here. As the script is intended to be also used directly/manually from tarballs extracted (anywhere).

Reply:
@michaelsembwever Check it now please.

Reply:
@adejanovski @michaelsembwever Can you check it, please.

Reply:
@adejanovski @michaelsembwever Can You check it, please.

Reply:
i realy want to see it in next cassandra reaper version